### ISSUE TITLE
[OpenMP] llvm-omp-device-info crashes

### ISSUE THREAD

**Issue by jdoerfert** [llvm/llvm-project#61392]
On a MI250x node, offloading works fine though:

```
llvm-omp-device-info                                                                                                                                                                                                
Device (0):
llvm-omp-device-info: /dev/shm/doerfert/rr/src/llvm-project/openmp/libomptarget/plugins-nextgen/common/PluginInterface/PluginInterface.h:774: GenericDeviceTy &llvm::omp::target::plugin::GenericPluginTy::getDevice(int32_t): Assertion `Devices[DeviceId] && "Device is unitialized"' failed.
[2]    999571 abort (core dumped)  llvm-omp-device-info
```

**Comment by josemonsalve2** [comment]
I'm trying to figure this out. It seems to be some initialization stuff of the new nextgen plug-in

**Comment by kevinsala** [comment]
Commit aa326559c443382030c735db167645685da245a2 fixes the crash and commit 843f496b71aece4097bcc780c3bb47bd0874844a adds the device information for AMDGPU (nextgen plugin).
